test/functorch/test_ops.py

```diff
@@ -351,7 +351,7 @@ def is_inplace(op, variant):
 
 vjp_fail = {
     xfail("tensor_split"),  # data_ptr composite compliance
-    # https://github.com/pytorch/pytorch/issues/96560
+    # Very minor accuracy issue on ROCm
```

Comment:
Why this change is needed?

Reply:
I wanted to decouple this skip reason from the quoted issue since we triaged it and found that it was just a very minor tolerance issue that does not require immediate attention. This way we can close the aforementioned issue.